src/ccmain/tesseractclass.h

```diff
@@ -897,6 +897,9 @@ class TESS_API Tesseract : public Wordrec {
   BOOL_VAR_H(tessedit_create_txt);
   BOOL_VAR_H(tessedit_create_hocr);
   BOOL_VAR_H(tessedit_create_alto);
+  BOOL_VAR_H(tessedit_create_page);
+  BOOL_VAR_H(tessedit_create_page_polygon);
+  BOOL_VAR_H(tessedit_create_page_wordlevel);
```

Comment:
I suggest to omit the prefix `tessedit_create` and simply use `page_xml_polygon` and `page_xml_wordlevel` (or `page_xml_level` with 0=line, 1=word which might allow a future implementation to add 2=glyph).

For hocr there exist already the parameters `hocr_font_info` and `hocr_char_boxes`, so the new parameters would fit better into that schema.